src/interpreter.rs

```diff
@@ -139,21 +140,32 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
                 struct Instance<'tcx>(DefId, &'tcx Substs<'tcx>);
                 impl<'tcx> fmt::Display for Instance<'tcx> {
                     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-                        ppaux::parameterized(f, self.1, self.0, ppaux::Ns::Value, &[],
-                            |tcx| tcx.lookup_item_type(self.0).generics)
+                        ppaux::parameterized(f,
+                                             self.1,
+                                             self.0,
+                                             ppaux::Ns::Value,
+                                             &[],
+                                             |tcx| tcx.lookup_item_type(self.0).generics)
                     }
                 }
-                err.span_note(span, &format!("inside call to {}", Instance(def_id, substs)));
+                err.span_note(span,
+                              &format!("inside call to {}", Instance(def_id, substs)));
             }
             err.emit();
         }
         r
     }
 
-    fn log<F>(&self, extra_indent: usize, f: F) where F: FnOnce() {
+    fn log<F>(&self, extra_indent: usize, f: F)
+        where F: FnOnce()
+    {
         let indent = self.stack.len() + extra_indent;
-        if !TRACE_EXECUTION { return; }
-        for _ in 0..indent { print!("    "); }
+        if !TRACE_EXECUTION {
+            return;
+        }
+        for _ in 0..indent {
+            print!("    ");
+        }
```

Comment:
For the for-loop... I don't know. But single-line return guards seem like a reasonable way to save a little vertical height.